docs/designs/conceptual-versioning.md

```diff
@@ -514,10 +512,12 @@ In current docfx v3, Toc file is build at the same time with `page` files, becau
 
 When build the toc file(including the toc file included in another), we have to cover:
 
-1. *Monikers* of every node in toc. We don't support setting monikers of node in toc file, 
+1. *Monikers* of every node in toc(We don't support setting monikers of node in toc file).
     1. If the node is a text node(`## Header`), the *monikers* of this node is the collection of its children's monikers.
     1. If the node is a link node(`## [Header](a.md)`), the *monikers* of this node is the collection of its children's monikers and the monikers of this node.
-1. *Monikers* of toc file. We don't support setting monikers of the toc file by config or file metadata, the *monikers* of toc file is the collection of every node's monikers.
+1. *Monikers* of toc file - Intersection of the monikerRange setting in the config and file metadata(If the intersection is empty, a warning will be logged).
```

Comment:
do you need add the case we talked offline?

Reply:
the test inside `moniker.yml` have covered this?

---
In reply to: [239359268](https://github.com/dotnet/docfx/pull/3725#discussion_r239359268) [](ancestors = 239359268)